samples/build-netlify/README.md

```diff
@@ -0,0 +1,25 @@
+# Build Netlify
+
+This Sidebar Extension shows how to use Netlify API so you could trigger Netlify builds manually while editing content.
+
+![Screenshot of extension](../../docs/assets/netlify-build.png "Screenshot of extension")
+
+## Requirements
+
+1. Get Build URL on [Netlify](https://www.netlify.com/docs/webhooks/).
+   * You can provide this information on real setup.
+2. (optional) If you want to see the status of the build you triggered, you need to provide the following information:
+   * Get [Personal Access Token](https://app.netlify.com/account/applications)
+   * Get API ID of your site in Site Details.
+   * **We would not recommend to expose this information on real setup, use only for demo purpose**
```

Comment:
IMPORTANT: In a real world application the token should be kept secret. A good approach would be to wrap the token on a Netlify Function or host the entire UI Extensions 3rd party so the server can handle the secret API token properly.